packages/react-components/components/KeyboardAwareScrollView.tsx

```diff
@@ -1,96 +0,0 @@
-import * as React from 'react'
-import {
-  EmitterSubscription,
-  findNodeHandle,
-  Keyboard,
-  Platform,
-  ScrollView,
-  ScrollViewProps,
-  TextInput,
-  UIManager,
-} from 'react-native'
-
-type ComponentOrHandle = null | number | React.Component<any, any> | React.ComponentClass<any>
-
-interface Props extends ScrollViewProps {
-  hasNavBar?: boolean
-}
-
-// Keeps the focused input into view when the keyboard is shown
-export default class KeyboardAwareScrollView extends React.Component<Props> {
```

Comment:
I agree it's annoying to bring in another dependency, but considering we are using it in some critical components and it causes crashes I think we should use the latest version of this library since it seems to be used and maintained pretty well. For example they recently added fixes for RN 0.63 as well as for iPhone X. Also the library itself is pretty small so I don't think it's a big increase to the app.

https://github.com/APSL/react-native-keyboard-aware-scroll-view/releases
https://github.com/APSL/react-native-keyboard-aware-scroll-view/blob/master/lib/KeyboardAwareHOC.js

Reply:
The problem is `react-native-keyboard-aware-scroll-view` doesn't work nicely with our `KeyboardSpacer` component and breaks the code input screen for instance:

<img src="https://user-images.githubusercontent.com/57791/101897815-6c753880-3bab-11eb-8dae-e450c1fc8ceb.png" width="50%" />

I built our `KeyboardSpacer` and `KeyboardAwareScrollView` to work together. Both are relatively simple and I wanted full control over them because `react-native-keyboard-aware-scroll-view` wasn't a good fit for some of our uses cases and didn't handle platform specific edge cases very well.

Of course we can revisit and recheck all of this, but I wouldn't do this in this PR.

Happy to jump back on a conversation to detail this more.

Reply:
Ah, I see it's more complicated than I initially thought. I will revert this and put in a null check for the crash and then we can revisit this later. Thanks Jean!